app/src/main/res/layout/proton_component_tracking_protection_panel.xml

```diff
@@ -0,0 +1,286 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Trying to figure out where this new layout is coming from? I don't see any details in the issue nor is it renamed from another file.

Reply:
Thanks for catching this it looks like it's a leftover of a local iteration.